src/drivers/gps/gps.cpp

```diff
@@ -511,6 +567,73 @@ int GPS::setBaudrate(unsigned baud)
 	return 0;
 }
 
+bool GPS::fileExist(const char *filename)
+{
+	struct stat buffer;
+	return stat(filename, &buffer) == 0;
```

Comment:
@bkueng: I'm out next week but I propose you try to avoid using this `stat` function and test it with Roman's or Nicolas' help. If it doesn't crash, and GPS still works, it can obviously go in.